### ISSUE TITLE
Two MQTT servers

### ISSUE THREAD

**Issue by pguich** [gluap/pyess#20]
Is it possible to publish the ess results to two MQTT servers?

I’m migrating my home assistant to another Raspberry, but I want to run both in parallel for a few hours to see if everything works before I turn off the old one. Pyess is running on an entirely different Raspberry.

btw, I’m extremely happy with pyess. Now if only the Wi-Fi connection on the ESS weren’t so unstable :(

**Comment by pguich** [comment]
thanks a lot, I'll try that.
I have another issue, I'll open one right now.
